lib/slurm_squeue_client.rb

```diff
@@ -120,20 +121,34 @@ def eligible_percent
   # @return [Integer] the total number of active/eligible jobs
   def available_jobs
     active_jobs + eligible_jobs
+  end 
+
+  # Total nodes available (idle) and total nodes used
+  #
+  # @return [Integer] the total number of idle/used jobs
+  def available_nodes
+    nodes_avail
+  end
+
+  # Total number of available and in use procs
+  #
+  # @return [Integer] the total number of procs 
+  def available_procs
```

Comment:
Use the number reported by sinfo as this calculation missed "other" proc states.  The output from `%C` contains the total:

```
$ sinfo -a -o "%C"
CPUS(A/I/O/T)
9661/9283/240/19184
```